testing/networks/80084/snapshots.md

```
@@ -2,15 +2,16 @@
-| Berachain Foundation - EU Hosted | https://storage.googleapis.com/bartio-snapshot-eu/index.html                                                                             | pebbledb  |
-| Berachain Foundation - Asia Hosted           | https://storage.googleapis.com/bartio-snapshot-as/index.html                                                          | pebbledb  |
-| Imperator                                            | [https://www.imperator.co/services/chain-services/testnets/bera-v2](https://www.imperator.co/services/chain-services/testnets/bera-v2)                                                   | goleveldb |
-| Blacknodes                                           | [https://services.blacknodes.net/Berachain-V2/](https://services.blacknodes.net/Berachain-V2/)                                                                                           | pebbledb  |
-| Contribution DAO                                     | [https://services.contributiondao.com/testnet/berachain-v2/snapshots](https://services.contributiondao.com/testnet/berachain-v2/snapshots)                                               | pebbledb  |
-| Staketab                                             | [https://services.staketab.org/docs/beacon-testnet/snapshot](https://services.staketab.org/docs/beacon-testnet/snapshot)                                                                 | pebbledb  |
-| A41                                                  | [https://berachain.snapshot.a41.io/](https://berachain.snapshot.a41.io)                                                                                                                  | pebbledb  |
-| Synergy Nodes                                        | [https://synergynodes.com/service/berachain-v2-testnet](https://synergynodes.com/service/berachain-v2-testnet)                                                                           | pebbledb  |
-| L0vd services                                        | [https://chain-services.l0vd.com/testnets/berachain_v2/snapshot#sync-from-snapshot-pebbledb](https://chain-services.l0vd.com/testnets/berachain_v2/snapshot#sync-from-snapshot-pebbledb) | pebbledb  |
+| Provider                                             | URL                                                                                                                                                                                                          | Database  |
+| ---------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------- |
+| Berachain Foundation                                 | [Link] (https://storage.googleapis.com/bartio-snapshot/index.html)                                                                                                                                           | pebbledb  |
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)